scripts/prepare_sector_network.py

```diff
@@ -2413,6 +2413,84 @@ def limit_individual_line_extension(n, maxext):
     hvdc = n.links.index[n.links.carrier == 'DC']
     n.links.loc[hvdc, 'p_nom_max'] = n.links.loc[hvdc, 'p_nom'] + maxext
 
+
+def apply_time_segmentation(n, segments, solver_name="cbc",
+                            overwrite_time_dependent=False):
```

Comment:
I would set the default to true. I think taking just the first value of clustered time steps would be less accurate.
```suggestion
                            overwrite_time_dependent=True):
```

Reply:
okay, we can do so. That would be different than how it is done in PyPSA-Eur